Dockerfile

```diff
@@ -1,17 +1,24 @@
-FROM alpine:edge
+FROM phusion/baseimage
```

Comment:
We should probably pin this to a specific tag of `phusion/baseimage` for good measure. Looks like 0.9.17 is latest right now.